src/tests/end-to-end/common/page.ts

```diff
@@ -18,8 +19,12 @@ export class Page {
         underlyingPage.on('pageerror', error => {
             forceEventFailure(`'pageerror' (console.error) with stack: ${error.stack}`);
         });
+        // Modifying this error handler as a workaround for #923
```

Comment:
nit: I think it makes sense to not have this comment with issue reference here.

Reply:
I'm not sure if you mean "you think it belongs next to the if statement instead" or "you don't think it's necessary at all". I agree with the former and strongly disagree with the latter.